app/src/components/projectSettings/ProjectUserTable.tsx

```diff
@@ -14,13 +14,13 @@ const ProjectUserTable = () => {
   const session = useSession().data;
 
   const isAdmin = useAccessCheck("requireIsProjectAdmin").access;
-
+  const owner = selectedProject?.projectUsers.find((u) => u.role === "OWNER");
   const utils = api.useContext();
 
   const canLeaveProject =
     !isAdmin ||
     !selectedProject ||
-    selectedProject.projectUsers.filter((u) => u.role === "ADMIN").length > 1;
+    selectedProject.projectUsers.filter((u) => u.role === "ADMIN" || u.role === "OWNER").length > 1;
```

Comment:
Can we drop this condition and replace the `!isAdmin` condition with `u.role === "OWNER"`?

Reply:
The previous logic around Admins leaving a project is a little different from the new logic around Owners leaving a project. Since all projects will have a single owner from now on, users who are admins should be able to leave at any time, and owners should not be able to leave at all.

This check to ensure that another admin was on a project before allowing the previous one to leave made sense because a project could have multiple admins, and needed at least one. Since each project now has one and only one admin, this check no longer serves that purpose.

Reply:
Yes, I understand the logic behind it.
There is a separate check that ensures that on owner cannot leave the project.
At the same time, there are no constraints on the database level that ensure that a project has an owner. We accidentally removed an owner by running some scripts. So this code ensures that at least one admin or owner should be associated with a project. 
Such cases are rare and noncritical. The question is whether we should ignore them to make the code look simpler.

Reply:
I think we should avoid deleting project owners, and assume that every project has an owner. If a project ends up without an owner through an error in our logic, billing will also be in a broken state.

We can and should avoid this edge case.

Reply:
I changed the code. It looks simpler if we do not think about such edge cases.